disperser/cmd/batcher/main.go

```diff
@@ -190,6 +190,11 @@ func RunBatcher(ctx *cli.Context) error {
 		if err != nil {
 			return err
 		}
+		sender, err := wallet.SenderAddress(context.Background())
```

Comment:
Don't have a developed perspective on this, but if this is just used for logging, should we error out the function if we can't get it?

Reply:
Yeah this value is actually used for validation: https://github.com/Layr-Labs/eigenda/blob/master/disperser/cmd/batcher/main.go#L210
I'll rebase this one as the logging got added in another PR